app/assets/stylesheets/application.scss

```diff
@@ -20,15 +20,17 @@ body {
   height:100%;
 }
 * {
-margin: 0px;
-padding: 0px;
+  margin: 0px;
+  padding: 0px;
 }
-  
+
+// @import "variables";
```

Comment:
this is commented out, maybe we don't need the _variable file after all?

Reply:
Oh yeah, good catch! It was trying to fix the whole bootstrap variable issue with `$enable-gradients: true;`, but it looks like it ended up getting fixed.